code/lib/preview-api/src/modules/preview-web/render/MdxDocsRender.ts

```diff
@@ -29,6 +29,8 @@ import { DocsContext } from '../docs-context/DocsContext';
 export class MdxDocsRender<TRenderer extends Renderer> implements Render<TRenderer> {
   public readonly type: RenderType = 'docs';
 
+  public readonly subtype = 'mdx';
```

Comment:
So I guess in this case, it would not emit a `COMPONENT_PREPARED` .. but does that mean that the store is in a perpetually unprepared state?